src/providers/backend.h

```diff
@@ -60,6 +60,9 @@ struct be_svc_data {
 
     struct be_svc_callback *callbacks;
     struct fo_server *first_resolved;
+
+    struct fo_server *current_resolved;
+    int current_resolved_attempts;
 };
```

Comment:
These name sounds weird, perhaps you meant `currently_resolved`? Wouldn't be `last_resolved_srv` better? Note that there is `last_good_srv`, perhaps it can be reused?

Reply:
The first one has been removed and the second has changed to `retry_attempts`